base.cfg

```diff
@@ -0,0 +1,115 @@
+[buildout]
+show-picked-versions = true
+extensions =
+    mr.developer
+
+parts =
+    instance
+    test
+# we use tox for testing and linting, by default
+#    code-analysis
+    coverage
+    test-coverage
+    createcoverage
+    releaser
+    i18ndude
+    omelette
+    robot
+    plone-helper-scripts
+    vscode
+
+develop = .
+
+abi-tag-eggs = true
+
+sources-dir = src-mrd
+
+[instance]
+recipe = plone.recipe.zope2instance
+user = admin:admin
+http-address = 8080
+environment-vars =
+    zope_i18n_compile_mo_files true
+eggs =
+    Plone
+    Pillow
+    plone.portlet.collection [test]
+
+[vscode]
+recipe = collective.recipe.vscode
+eggs = ${instance:eggs}
+autocomplete-use-omelette = True
+
+# [code-analysis]
+# recipe = plone.recipe.codeanalysis
+# directory = ${buildout:directory}/src/ukd
+# return-status-codes = False
+
+
+[omelette]
+recipe = collective.recipe.omelette
+eggs = ${instance:eggs}
+
+
+[test]
+recipe = zc.recipe.testrunner
+eggs = ${instance:eggs}
+initialization =
+    os.environ['TZ'] = 'UTC'
+defaults = ['-s', 'plone.portlet.collection', '--auto-color', '--auto-progress']
+
+
+[coverage]
+recipe = zc.recipe.egg
+eggs = coverage
+
+
+[test-coverage]
+recipe = collective.recipe.template
+input = inline:
+    #!/bin/bash
+    export TZ=UTC
+    ${buildout:directory}/bin/coverage run bin/test $*
+    ${buildout:directory}/bin/coverage html
+    ${buildout:directory}/bin/coverage report -m --fail-under=90
+    # Fail (exit status 1) if coverage returns exit status 2 (this happens
+    # when test coverage is below 100%.
+output = ${buildout:directory}/bin/test-coverage
+mode = 755
+
+
+[createcoverage]
+recipe = zc.recipe.egg
+eggs = createcoverage
+
+
+[robot]
```

Comment:
There are no robot tests, so we do not need this.

Reply:
@mauritsvanrees ok, i will be remove this